packages/element-library/src/image/test/output.js

```diff
@@ -25,6 +25,8 @@ import { renderToStaticMarkup } from '@googleforcreators/react';
 import ImageOutput from '../output';
 
 describe('Image output', () => {
+  jest.retryTimes(3, { logErrorsBeforeRetry: true });
```

Comment:
Why is this here and not at the start of the file?

Reply:
See answer in other thread. It doesn‘t need to be at the top of the file.